mailpoet/lib/API/JSON/v1/Tags.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\API\JSON\v1;
+
+use MailPoet\API\JSON\Endpoint as APIEndpoint;
+use MailPoet\Entities\TagEntity;
+use MailPoet\InvalidStateException;
+use MailPoet\Tags\TagRepository;
+
+class Tags extends APIEndpoint {
+
```

Comment:
Maybe we should also add here something like:

```php
  public $permissions = [
    'global' => AccessControl::PERMISSION_MANAGE_TAGS,
  ];
```

Reply:
I agree in principal. But I worry how we ensure over time the `PERMISSION_MANAGE_AUTOMATIONS` permission and `PERMISSION_MANAGE_TAGS` permission do not diverge (one could argue that also for the current `PERMISSION_MANAGE_SETTINGS` and `PERMISSION_MANAGE_AUTOMATIONS`)

Reply:
@websupporter Yeah, true, but maybe we should do it for consistency? Or should we keep settings? 🤔 I guess we don't differentiate these at the moment anyway...